.flake8

```diff
@@ -0,0 +1,5 @@
+[flake8]
```

Comment:
Can we add some comments to what the `ignore` and `select` values are? I think adding at least a short description on what the ignored items are would go a long way, it looks little cryptic otherwise.

Reply:
I've massively simplified this file, it now only ignores E501, which is a PEP8 line length error and is handled by the `max-line-length` field anyway. 

My thinking is that we can add to the ignore list if we find any rules that we're regularly violating (for good enough reasons) but we can get by with the default set to begin with.